src/components/Data/Collections/Update.vue

```diff
@@ -53,10 +53,13 @@ export default {
         this.collectionName
       )
     },
-    mappingAttributes() {
-      return this.collection
-        ? omit(this.collection.mapping, '_kuzzle_info')
-        : null
+    getFullMappings() {
```

Comment:
```suggestion
    fullMappings() {
```
Nitpicking: a computed function is always a getter function, you don't need to specify a verb in its name